src/components/Stories/README.md

```diff
@@ -0,0 +1,47 @@
+## Stories
+
+Component for displaying stories.
```

Comment:
Instagram user will have different meaning for stories. It is better to explain what it is really is.

It is basically carousel in modal

Reply:
I will add more info to readme